fast_neural_style/download_saved_models.py

```diff
@@ -12,7 +12,10 @@
 try:
     from torch.utils.model_zoo import _download_url_to_file
 except ImportError:
-    from torch.hub import _download_url_to_file
+    try:
+        from torch.hub import download_url_to_file as _download_url_to_file
+    except ImportError:
+        from torch.hub import _download_url_to_file
```

Comment:
`_download_url_to_file` has been renamed to `download_url_to_file` (made public with no leading `_`)